utils/prune_binaries.py

```diff
@@ -31,10 +31,11 @@
     'third_party/dawn/tools/golang/',
     'third_party/depot_tools/external_bin/',
     'third_party/devtools-frontend/src/third_party/esbuild/',
-    'third_party/google-java-format/'
```

Comment:
Shouldn't it error out if it failed to remove anything? To make it easier to detect errors like this? (It seems like this fixed an issue where it was trying to prune `third_party/google-java-format/third_party/libei/`)

Reply:
The pruning list will error out when a path doesn't exist, so the contingent paths were set up specifically to be pruned only *if* they exist without throwing errors.  But you're right that there should at least be some sort of output that allows us to catch problems.  I'll experiment with adding info-level output for the contingent paths.